packages/flutter_tools/lib/src/tester/flutter_tester.dart

```diff
@@ -151,11 +150,14 @@ class FlutterTesterDevice extends Device {
       return LaunchResult.failed();
     }
 
-    final String assetDirPath = _fileSystem.path.join(_buildDirectory, 'flutter_assets');
+    final Directory assetDirectory = _fileSystem.systemTempDirectory
+      .createTempSync('flutter-tester')
+      ..createSync(recursive: true);
```

Comment:
Why the double create?

Reply:
I might have always misunderstood the tempSync method - I though it only "created" the directory name but did not actually create the directory..

Reply:
Fixed

Reply:
If it doesn't create it, and only returns the name, that would be a bug since it would break atomicity.

Reply:
Good point :)

Reply:
(assuming it worked correctly though....)